app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PagesController < ApplicationController
+  def home
+    render :home
```

Comment:
There is no need to place this line in any of the methods. Rails does it for you.
Please remove those from all the methods and verify that those pages still work 😄